.github/workflows/repo.hex.pm.yml

```
@@ -61,10 +75,37 @@ jobs:
                   aws s3 cp "${tarball}" "s3://${{ env.AWS_S3_BUCKET }}/docs/${tarball}" \
                       --cache-control "public,max-age=3600" \
                       --metadata "{\"surrogate-key\":\"${surrogate_key}\",\"surrogate-control\":\"public,max-age=604800\"}"
-                  purge_key "${surrogate_key}"
-                  sleep 2
-                  purge_key "${surrogate_key}"
-                  sleep 2
-                  purge_key "${surrogate_key}"
+                  purge "${surrogate_key}"
               fi
           done
+      - name: Upload Precompiled to S3
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.HEX_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.HEX_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: ${{ secrets.HEX_AWS_REGION }}
+          AWS_S3_BUCKET: ${{ secrets.HEX_AWS_S3_BUCKET }}
+        run: |
+          mv elixir-otp-${{ matrix.otp }}.zip elixir.zip
+          version=$(echo ${{ github.ref_name }} | sed -e 's/^v//g')
+          aws s3 cp elixir.zip "s3://${{ env.AWS_S3_BUCKET }}/builds/elixir/${version}${{matrix.otp}}.zip" --cache-control "public,max-age=3600" --metadata '{"surrogate-key":"builds","surrogate-control":"public,max-age=604800"}'
```

My understanding is that this is the correct URL:

```suggestion
          aws s3 cp elixir.zip "s3://${{ env.AWS_S3_BUCKET }}/builds/elixir/${{github.ref_name}}-otp-${{matrix.otp}}.zip" --cache-control "public,max-age=3600" --metadata '{"surrogate-key":"builds","surrogate-control":"public,max-age=604800"}'
```

/cc @ericmj

---

I am glad that you mentioned it 😅, waiting for @ericmj ☺️

---

I managed to find a way to curl builds.txt via `curl https://repo.hex.pm/builds/elixir/builds.txt`  😄